README.md

```diff
@@ -1,15 +1,51 @@
+<div id="top"></div>
+
 # HASH
 
 This repository contains HASH's open-source and public code, documentation, and other key resources, including:
 
-- `packages/hash`: The codebase for HASH ([Info](https://hash.ai/platform/index)) - coming soon, available under an Apache 2.0 compatible license
+- `packages/hash`: The codebase for HASH - an open-source, data-centric workspace based on the [Block Protocol](https://github.com/blockprotocol/blockprotocol), coming soon
 - `packages/engine`: The codebase for [hEngine](packages/engine) ([Info](https://hash.ai/platform/engine)) - available under the Server Side Public License
 - `packages/engine/stdlib`: The [standard library](packages/engine/stdlib) of helper functions available in all HASH simulations
 - `resources/docs`: A user guide to the whole HASH platform ([hash.ai/docs](https://hash.ai/docs))
 - `resources/glossary`: A glossary of terms explaining common concepts relevant to the use of HASH ([hash.ai/glossary](https://hash.ai/glossary))
 
-Please see:
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+
```

Comment:
Looks like we don’t run Prettier for top-level MD files. Otherwise, CI would fail for the PR because of several newlines. I guess we can fix this later, just something to be aware of.

Reply:
Captured [here](https://app.asana.com/0/1199540314699189/1201594305573233/f) for followup.